src/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import { Meta } from '@storybook/react';
+import Toast from './Toast';
+import { ToastProvider } from './ToastContextProvider';
+import Button from '../Button/Button';
+import { useToast } from './useToast';
+import { ButtonVariants } from '../Button/types';
+
+const meta = {
+  title: 'Components/Toast',
+  component: Toast,
+  decorators: [(Story) => <ToastProvider>{Story()}</ToastProvider>],
```

Comment:
should this be wrapped globally in case our other components need it in their stories?